### PR TITLE
Generate did identities

### PR DESCRIPTION
**Pull request by dangershony**
close https://github.com/block-core/blockcore-dns/issues/11

### CODE REVIEW

File: src/Blockcore.Dns/appsettings.json [hunk 1]
```diff
@@ -21,10 +21,12 @@
   "Dns": {
     "ListenPort": "53",
     "IntervalMin": "5",
-    "EndServerIp": "192.168.1.1"
+    "EndServerIp": "192.168.1.1",
+    "Identities": [
+      "did:is:03e9bcd2f568bddec7c2f36132ce4017e6a98077a9bc4ad7105c48b12f7b9ec7cb,",
```
Comment (sondreb) [inline src/Blockcore.Dns/appsettings.json:26]: There is a bug here, an extra "," added at the end.

**Review by sondreb — COMMENTED** [review]
Minor feedback

File: src/Blockcore.Dns/appsettings.json (hunk 1 above)
Comment (dangershony) [reply src/Blockcore.Dns/appsettings.json:26]: ah ok well spotted, this is of course a fake did but its still a good example

**Comment by dangershony** [comment]
Until this PR is published we will use ECDSA sigs 
https://github.com/block-core/blockcore/pull/404

**Pull request merged**